cli/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ _Released 03/1/2023 (PENDING)_
 
 **Breaking Changes:**
 
+- The [`.should()` and `.and()`](/api/commands/should) assertions are now queries. This means that they are fully retryable, and it is safe to chain further commands that interact with the DOM after them. Assertions are also now re-run as part of aliases. Addresses [#25134](https://github.com/cypress-io/cypress/issues/25134) and [#25491](https://github.com/cypress-io/cypress/issues/25491). Addressed in [#25738](https://github.com/cypress-io/cypress/pull/25738).
```

Comment:
```suggestion
- The [`.should()`](https://docs.cypress.io/api/commands/should) and [`.and()`](/api/commands/and) assertions are now queries. This means that they are fully retryable, and it is safe to chain further commands that interact with the DOM after them. Assertions are also now re-run as part of aliases. Addresses [#25134](https://github.com/cypress-io/cypress/issues/25134) and [#25491](https://github.com/cypress-io/cypress/issues/25491). Addressed in [#25738](https://github.com/cypress-io/cypress/pull/25738).
```

> This means that they are fully retryable, and it is safe to chain further commands that interact with the DOM after them. 

I don't know what this means and if users will either since assertions were documented such that they retried to pass.